src/PlaneWaveBasis.jl

```diff
@@ -24,9 +26,13 @@ end
 
 
 """
-The list of G vectors of a given `basis` or `kpoint`.
+The list of G vectors of a given `basis` or `kpoint`, in reduced coordinates.
 """
 G_vectors(kpt::Kpoint) = kpt.G_vectors
+"""
+The list of G vectors of a given `basis` or `kpoint`, in cartesian coordinates.
+"""
+G_vectors_cart(kpt::Kpoint) = (kpt.model.recip_lattice * G for G in G_vectors(kpt))
```

Comment:
I don't like having another function for this. I would prefer `G_vectors(abc, cartesian=true)` or `G_vectors(abs, :cartesian)`

Reply:
What is the point apart from being more verbose?

Reply:
I don't like to abbreviate it as `cart` and it is one function (ok weak point).

Reply:
Why? cart and coord are standard, unmistakeable abbreviations. The one function I would buy if there was more than two coordinates sytems, but it just seems more trouble than not

Reply:
ok fine, convinced.